### PR TITLE
Added web block for countdown timer.

### PR DESCRIPTION
**Pull request by leocsdev**
# Zero To Mastery - Webblocks 2022 📦📦

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
- Added web block for count-down timer. It allows the user to enter the count down name and time to begin the count down.

## Breaking changes
- None

## Screenshots
icon

### CODE REVIEW

File: src/blocks/leocsdev4/index.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    )
+    let minutes = Math.floor((distance % (1000 * 60 * 60)) / (1000 * 60))
+    let seconds = Math.floor((distance % (1000 * 60)) / 1000)
+
+    // Output the result in an element with id="countdown-timer"
+    document.querySelector('#countdown-timer').innerHTML =
+      days + 'd ' + hours + 'h ' + minutes + 'm ' + seconds + 's '
+
+    console.log('x:', x)
+    console.log('distance:', distance)
```
Comment (sirnotes) [inline src/blocks/leocsdev4/index.js:57]: Please remove debugging code :)

It's generally not a good idea to include debugging code in the main line code (production ready).

**Review by sirnotes — CHANGES_REQUESTED** [review]

File: src/blocks/leocsdev4/index.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+      document.querySelector('#countdown-timer').innerHTML = 'EXPIRED'
+      formSetTime.style.display = 'block'
+      btnClear.style.display = 'none'
+    }
+  }, 1000)
+}
+
+// Reset - Just reload the page for simplicity
+btnClear.addEventListener('click', function () {
+  console.log('clear')
```
Comment (sirnotes) [inline src/blocks/leocsdev4/index.js:73]: - debugging code

File: src/blocks/leocsdev4/index.js [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+let dateTimeValue = ''
+let titleValue = ''
+
+formSetTime.addEventListener('submit', function (e) {
+  e.preventDefault()
+
+  dateTimeValue = inputDateTime.value
+  titleValue = inputTitle.value
+
+  console.log('dateTime', dateTimeValue)
```
Comment (sirnotes) [inline src/blocks/leocsdev4/index.js:17]: - debugging code

File: src/blocks/leocsdev4/index.js (hunk 1 above)
Comment (leocsdev) [reply src/blocks/leocsdev4/index.js]: Removed now. Thanks

File: src/blocks/leocsdev4/index.js (hunk 2 above)
Comment (leocsdev) [reply src/blocks/leocsdev4/index.js]: Removed now. Thanks

File: src/blocks/leocsdev4/index.js (hunk 3 above)
Comment (leocsdev) [reply src/blocks/leocsdev4/index.js]: Removed now. Thanks

**Review by sirnotes — APPROVED** [review]

**Comment by sirnotes** [comment]
Thank you!

**Pull request merged**